frontend/uddhar-frontend/src/authentication/pages/Login.jsx

```
@@ -90,22 +88,17 @@ function Login() {
+            {errors.password && <div className="text-red-500"> {errors.password}</div>}
             <div className="flex flex-col gap-1 md:gap-5">
               <button
                 type="submit"
-                className="w-2/3  self-center rounded-md bg-amber-700 text-white text-sm md:text-xl mt-2 py-2"
+                className="w-2/3  self-center rounded-md bg-amber-500 hover:bg-amber-700 text-white text-sm md:text-xl mt-2 py-2 cursor-pointer"
               >
                 Login
               </button>
-              <button className="w-full text-blue-400 font-bold text-sm md:text-lg ">
-               <a href="/password-recovery">Forget password?</a>
+              <button className="w-full text-blue-400 font-bold text-sm md:text-lg cursor-pointer hover:text-blue-700" onClick={()=> navigate('/password-recovery')}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)